docs/GetStarted/getting_started_linux_bazel.md

```diff
@@ -31,11 +32,10 @@ We recommend Clang. GCC is not fully supported.
 $ sudo apt install clang
 ```
 
-Set environment variables:
+### Install python3 numpy
 
 ```shell
-export CC=clang
-export CXX=clang++
```

Comment:
Isn't it needed to specify the compiler any longer? Haven't tested, but I assume these lines are still required.

Reply:
This was brought up by Lei [here](https://github.com/google/iree/pull/2201#discussion_r440153349). Basically people should now be able to configure their environment by running `python3 configure_bazel.py`. This has the side effect of introducing a `python3` `numpy` dependency.

Reply:
Oh, I missed that one. Excuse the noise.